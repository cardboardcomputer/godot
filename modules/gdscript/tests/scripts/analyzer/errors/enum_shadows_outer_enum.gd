enum { V }

class InnerClass:
	enum { V }

func test():
	pass
