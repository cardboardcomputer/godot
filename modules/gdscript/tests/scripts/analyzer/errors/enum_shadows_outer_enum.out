GDTEST_PARSER_ERROR
Name "V" is already used as a class enum value.
